Source/ProceduralPart.cs

```diff
@@ -1452,6 +1452,17 @@ public void PartParentChanged(Part newParent)
 				childToParent = part.FindAttachNodeByPart(newParent);
 			}
 
+            bool srfAttached = false;
+			if (childToParent == null && newParent != null)
```

Comment:
This code looks a bit weird, are you using tabs here?

Reply:
True that, looks fine in IDE, didn't notice in git diff output.